data/campaigns/Heir_To_The_Throne/scenarios/20b_Underground_Channels.cfg

```diff
@@ -268,4 +279,400 @@
+#
+# I chose these effects because we're close to the end, the next scenario (S22) has some large forests but,
+# thereafter, there is not a lot of use for forest skills, since most of theterritory is open grasslands.
+# So these effects should not have too large an effect upon the remaining parts of the campaign.
+#
+# GL-2016JUL
+
+#po Haralamdum is ancient, and his mind does not work well any more. He takes things very slowly and carefully, even for a Wose. He halts and pauses, often searching for something in his memory, using meaningless syllables to fill the gap. The syllables of his name come from the phrase he uses to mean "that is all" or "I am done" which he says often, usually out of frustration with how hasty everyone seems. Nobody alive knows his real name, and he has likely forgotten as well. Feel free to add, move, or remove pauses to break up, or even scatter, his train of thought. For inspiration, think of Asperger Syndrome.
+
+#po For background references, see Delfador's Memoirs, scenario 8 ("Ur Thorodor").
+
+    [event]
+        name=prestart
+
+        {VARIABLE met_wose no}
+    [/event]
+
+    [event]
+        name=victory
+
+        {CLEAR_VARIABLE book_taken,met_wose}
+    [/event]
+
+    [event]
+        name=die
+        [filter]
+            id=Haralamdum
+        [/filter]
+
+        [message]
+            speaker=narrator
+            image="wesnoth-icon.png"
+            message= _ "From far in the distance comes the sound of branches snapping, followed by a long, hollow boom. The world goes silent for a moment."
+        [/message]
+
+        [message]
+            speaker=Kalenz
+            message= _ "Can you feel it? It is as if the entire forest is stunned! Something has happened and, I fear, not for the good."
+        [/message]
+    [/event]
+
+    [event]
+        name=die
+        [filter]
+            id=Muff Argulak
+        [/filter]
+
+        [message]
+            speaker=narrator
+            image="wesnoth-icon.png"
+            message= _ "A hollow screech echos throughout the caverns. The air suddenly seems fresher and lighter."
+        [/message]
+
+        [if]
+            [not]
+                {CHECK_VARIABLE second_unit.side 1}
+            [/not]
+            [then]
+                [message]
+                    speaker=Delfador
+                    message= _ "Whatever that was, the world rejoices."
+                [/message]
+            [/then]
+        [/if]
+    [/event]
+
+    [event]
+        name=sighted
+        [filter]
+            side=4
+        [/filter]
+        [filter_second]
+            side=1
+        [/filter_second]
+        [filter_condition]
+            [have_unit]
+                id=Muff Argulak
+            [/have_unit]
+            [have_unit]
+                id=Haralamdum
+            [/have_unit]
+        [/filter_condition]
+
+        [if]
+            {CHECK_VARIABLE unit.id Haralamdum}
```

Comment:
Wait... `CHECK_VARIABLE`? Is that different from `VARIABLE_CONDITIONAL` somehow?